breathecode/jobs/tests/actions/test_fetch_sync_all_data.py

```diff
@@ -0,0 +1,193 @@
+        result = fetch_sync_all_data(model.spider)
+        self.assertEqual(result, DATA)
+        self.assertEqual(requests.get.call_args_list, [
+            call('https://app.scrapinghub.com/api/jobs/list.json',
+                 params=(
+                     ('project', model.zyte_project.zyte_api_deploy),
+                     ('spider', model.zyte_project.platform.name),
+                     ('state', 'finished'),
+                 ),
+                 auth=(model.zyte_project.zyte_api_key, ''))
+        ])
+
+    """
+    🔽🔽🔽 With two Spiders
+    """
+
+    @patch(REQUESTS_PATH['get'],
+           apply_requests_get_mock([(200, 'https://app.scrapinghub.com/api/jobs/list.json', DATA)]))
+    def test_fetch_funtion__with_two_spiders(self):
+        """Test /answer With two Spiders"""
+        import requests
+
+        model_1 = self.generate_models(spider=True)
+        model_2 = self.generate_models(spider=True)
+        result_1 = fetch_sync_all_data(model_1.spider)
+        result_2 = fetch_sync_all_data(model_2.spider)
+
+        self.assertEqual(result_1, DATA)
+        self.assertEqual(result_2, DATA)
+        self.assertEqual(requests.get.call_args_list, [
+            call('https://app.scrapinghub.com/api/jobs/list.json',
+                 params=(
+                     ('project', model_1.zyte_project.zyte_api_deploy),
+                     ('spider', model_1.zyte_project.platform.name),
+                     ('state', 'finished'),
+                 ),
+                 auth=(model_1.zyte_project.zyte_api_key, '')),
+            call('https://app.scrapinghub.com/api/jobs/list.json',
+                 params=(
+                     ('project', model_2.zyte_project.zyte_api_deploy),
+                     ('spider', model_2.zyte_project.platform.name),
+                     ('state', 'finished'),
+                 ),
+                 auth=(model_2.zyte_project.zyte_api_key, ''))
+        ])
+
+    """
+    🔽🔽🔽 Verify fetch.function.was.calletd with.one.Spider
+    """
+
+    @patch(REQUESTS_PATH['get'],
+           apply_requests_get_mock([(200, 'https://app.scrapinghub.com/api/jobs/list.json', DATA)]))
+    def test_verify_fetch_funtions_was_calletd(self):
+        """Test /Verify fetch.function.was.calletd with.one.Spider"""
+        import requests
+
+        model = self.generate_models(spider=True)
+        result = fetch_sync_all_data(model.spider)
+        requests.get.assert_called()
+        self.assertEqual(result, DATA)
+
+    """
+    🔽🔽🔽 status ok.fetch.to.api
+    """
+
+    @patch(REQUESTS_PATH['get'],
+           apply_requests_get_mock([(200, 'https://app.scrapinghub.com/api/jobs/list.json', {
+               'status': 'ok',
+               'data': []
+           })]))
+    def test_status_ok_fetch_to_api(self):
+        """Test /status ok.fetch.to.api"""
+        import requests
+
+        model = self.generate_models(spider=True)
+        result = fetch_to_api(model.spider)
```

Comment:
Is iscorrect test the funcion `fetch_to_api` here because this test is avoid `fetch_sync_all_data`